WMF Framework/ImageCacheHeaderProvider.swift

```diff
@@ -0,0 +1,49 @@
+
+import Foundation
+
+class ImageCacheHeaderProvider: CacheHeaderProviding {
+    func requestHeader(url: URL, forceCache: Bool = false) -> [String: String] {
+        var header: [String: String] = [:]
+        
+        guard let itemKey = itemKeyForURL(url) else {
+            return header
+        }
+        
+        header[Session.Header.persistentCacheItemKey] = itemKey
+        
+        if let variant = variantForURL(url) {
+            header[Session.Header.persistentCacheItemVariant] = variant
+        }
+        
+        header[Session.Header.persistentCacheItemType] = "Image"
+        header[Session.Header.persistentCacheForceCache] = String(forceCache)
+        //The accept profile is case sensitive https://gerrit.wikimedia.org/r/#/c/356429/
+        header["Accept"] = "application/json; charset=utf-8; profile=\"https://www.mediawiki.org/wiki/Specs/Summary/1.1.2\""
```

Comment:
This header isn't necessary for image requests - it tells the server we want UTF8 encoded JSON and the specific spec to use for the endpoint. Looking closely, I think I've used this incorrectly elsewhere, it should only go to the summary endpoint